package.json

```diff
@@ -44,25 +44,25 @@
   "devDependencies": {
     "bookshelf-jsdoc-theme": "^1.0.0",
     "chai": "4.0.2",
-    "eslint": "^5.10.0",
+    "eslint": "^5.16.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-prettier": "^3.0.0",
     "husky": "^2.4.1",
     "istanbul": "^0.4.5",
     "jsdoc": "^3.4.0",
-    "knex": "^0.17.0",
+    "knex": "0.17.6",
```

Comment:
We can't use 0.19.+ unless we drop Node 6 from Travis.

Reply:
We dropped support from Travis.